### PR TITLE
[TW-1074] Update Keen integration doc

### PR DESCRIPTION
**Pull request by akinard-postman**
This PR makes the following updates:
- Removed the screenshots from steps 1 through 3. This may help reduce some maintenance burden and did not seem necessary (and made the page longer).
- Updated some of the UI element references.
- Removed old/unused frontmatter.
- Minor wording update to the second intro paragraph.

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/keen.md [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
+You can use Keen IO for API-based computation, Amazon S3 backups, and building visualizations and dashboards for your APIs. Connect your Postman Monitor results to Keen Streams with the Postman Keen integration.
 
-Setting up a Keen integration requires you to get a project ID and API key before configuring Postman Monitors to push to Keen streams. After you set up the integration, you can view all monitoring data in Keen IO.
+Setting up a Keen integration requires a project ID and API key before configuring Postman Monitors to push to Keen streams. After you set up the integration, you can view all monitoring data in Keen IO.
 
 ## Retrieving your Keen IO project ID and API Key
 
 1. Sign in to [Keen IO](https://keen.io/users/login/).
 1. Select **Projects**.
-
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/available-integrations/keen.md:17]: I recommend always having a blank line above and below all screenshots--that way you'll get appropriate spacing around the image in the rendered content

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! Nice refresh! 😀

File: src/pages/docs/integrations/available-integrations/keen.md (hunk 1 above)
Comment (akinard-postman) [reply src/pages/docs/integrations/available-integrations/keen.md:17]: Applied that update (and the image update I forgot 🤦‍♀️ ) in commit 64214860997d19939bfc438c09347e59747f0a4c

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

**Pull request merged**